tests/geometry/test_tools.py

```diff
@@ -542,13 +546,17 @@ def naughty_function_fallback(wire, var=1, *, var2=[1, 2], **kwargs):
         raise _FallBackError(result=result)
 
 
-class TestLogFailedGeometryOperation:
+class TestLogFailedGeometryOperationSerialisation:
 
     wires = [
         make_polygon({"x": [0, 2, 2, 0], "y": [-1, -1, 1, 1]}, closed=True),
         make_circle(),
         make_ellipse(),
         PrincetonD().create_shape(),
+        PolySpline().create_shape(),
+        PictureFrame().create_shape(),
+        # TODO: Fix serialisation for this (origin uncertain)
```

Comment:
Spun off to #965 ; could be a wee bit of a rabbit hole and I'd like to crack on with #895